pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu

```diff
@@ -100,8 +103,9 @@ RUN mkdir /tmp/efa \
   && rm -rf /tmp/efa \
   && rm -rf /tmp/aws-efa-installer-${EFA_VERSION}.tar.gz
 
+RUN echo "pml = ob1" >> /opt/amazon/openmpi/etc/openmpi-mca-params.conf
```

Comment:
nit: [ADDRESS LATER] Use `OPEN_MPI_PATH` because we have already assigned `/opt/amazon/openmpi` to an ARG.